Classes/Parsing/OrgFileParser.m

```diff
@@ -86,6 +86,9 @@ - (void)parse {
         NSString *line;
         NSMutableString *bodyBuffer = [[NSMutableString alloc] init];
         int lastNumStars = 0;
+        // meta counter for last heading
+        // because lastNumStars will be resetted underway
+        int veryLastNumStars = 0;
```

Comment:
This might be more clearly named as something like `countLastNumStars`, it's not clear to me what `very` on the var name means.